src/StackAdmin/Azs.Compute.Admin/Tests/src/Quota.Tests.ps1

```diff
@@ -69,7 +69,7 @@ InModuleScope Azs.Compute.Admin {
 
                 # Subscriber Usage Aggregate
                 $Quota.AvailabilitySetCount | Should Not Be $null
-                $Quota.CoresLimit           | Should Not Be $null
+                $Quota.CoresCount           | Should Not Be $null
                 $Quota.VirtualMachineCount  | Should Not Be $null
                 $Quota.VmScaleSetCount      | Should Not Be $null
```

Comment:
You are not validating the new Quota properties.  Add validation for those.

Reply:
done